app/Http/Requests/Legacy/Two/PostRequest.php

```diff
@@ -31,7 +31,7 @@ public function rules()
             'caption' => 'nullable|string',
             'status' => 'in:pending,accepted,rejected',
             'source' => 'nullable|string',
-            // 'file' => 'string', // @TODO - should do some better validation around files.
+            'file' => 'image|dimensions:min_width=400,min_height=400,max_width=5000,max_height_4000',
```

Comment:
haha that `@TODO` is damning.

Reply:
Does the `image` and `dimensions` validator work if we're uploading a data URL string here (instead of a legit file, like we do on v3)? I think that's why we originally didn't do this.

Reply:
@DFurnes tested with a base64 string and validation worked!